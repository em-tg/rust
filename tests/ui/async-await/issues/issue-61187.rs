//@ check-pass
//@ edition:2018

fn main() {}

async fn response(data: Vec<u8>) {
    data.reverse(); //~ WARNING E0596
}
