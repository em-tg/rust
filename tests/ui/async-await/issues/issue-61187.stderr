warning[E0596]: cannot borrow `data` as mutable, as it is not declared as mutable
  --> $DIR/issue-61187.rs:7:5
   |
LL |     data.reverse();
   |     ^^^^ cannot borrow as mutable
   |
   = note: `#[warn(mut_non_mut)]` on by default
help: consider changing this to be mutable
   |
LL | async fn response(mut data: Vec<u8>) {
   |                   +++

warning: 1 warning emitted

For more information about this error, try `rustc --explain E0596`.
